test/prompt/conftest.py

```diff
@@ -10,3 +15,27 @@ def mock_supports(model_name_or_path, **kwargs):
         return model_name_or_path == model_name
 
     return Mock(**{"model_name_or_path": model_name, "supports": mock_supports, "invoke.return_value": response})
+
+
+@pytest.fixture
+def prompt_model(request, haystack_azure_conf):
+    if request.param == "openai":
+        api_key = os.environ.get("OPENAI_API_KEY", "KEY_NOT_FOUND")
+        if api_key is None or api_key == "":
```

Comment:
We set the `"KEY_NOT_FOUND"` default for `api_key` so it can't be None.
Let's simplify the if statement:
```python
if api_key == "":
            api_key = "KEY_NOT_FOUND"
```
if you want the exact same behavior. However, why not remove the if block completely?